libpod/events/nullout.go

```diff
@@ -20,3 +20,8 @@ func NewNullEventer() Eventer {
 	e := EventToNull{}
 	return e
 }
+
+// String returns a string representation of the logger
+func (e EventToNull) String() string {
+	return "devnull"
```

Comment:
We are now referring to this as "none". I believe so it should be repored to the user as such.